.vib/azure-cli/goss/azure-cli.yaml

```diff
@@ -0,0 +1,7 @@
+command:
+  check-python-module:
+    exec: /opt/bitnami/azure-cli/venv/bin/python -c 'import azure.cli'
+    exit-status: 0
+  check-cli-help:
+    exec: az --help
+    exit-status: 0
```

Comment:
I think this check can be omitted given that you are already checking the binary usage with `az --version`, wdyt?